service/geopmdpy/access.py

```diff
@@ -28,15 +29,18 @@ def __init__(self, geopm_proxy):
                     dasbus proxy for the GEOPM D-Bus interface.
 
         """
-        try:
-            geopm_proxy.PlatformGetGroupAccess
-        except DBusError as ee:
-            if 'io.github.geopm was not provided' in str(ee):
-                err_msg = """The geopm systemd service is not enabled.
+        if geopm_proxy is None:
+            self._access_lists = system_files.AccessLists()
+        else:
+            try:
+                geopm_proxy.PlatformGetGroupAccess
```

Comment:
This looks surprising to me since it's not making a function call. But it was already working before. Is PlatformGetGroupAccess an attribute that makes a dbus call in its accessor's implementation?

Reply:
This is the best way I could make trigger an exception without *calling* one of our methods.  We don't have a "Hello" method or the like.